nominees/mochcare.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "MOCHCare",
+  "aliases": [
+    "MLC"
```

Comment:
I am not sure how this alias connects to the project. Recommend removing.

Reply:
Fix in commit 13090a8